packages/legacy/core/App/App.tsx

```diff
@@ -25,11 +25,10 @@ import { defaultConfiguration } from './defaultConfiguration'
 import { initLanguages, initStoredLanguage, translationResources } from './localization'
 import RootStack from './navigators/RootStack'
 import { theme } from './theme'
-//import { credentialOfferTourSteps, credentialsTourSteps, proofRequestTourSteps } from './index'
 
-initLanguages(translationResources)
+function App(system: Container) {
```

Comment:
Where possible can we fix these to be fat arrow `=>` functions?

Reply:
done